app/src/test/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsViewModelTest.kt

```diff
@@ -230,10 +242,12 @@ class TransactionDetailsViewModelTest {
     }
 
     @Test
+    @Ignore
     fun `submitConfirmation (successful) emits ConfirmationSubmitted`() = runBlockingTest {
```

Comment:
same here

Reply:
Fixed in 6089571d0ad6e5fb42425ccec8725f310a0904c0